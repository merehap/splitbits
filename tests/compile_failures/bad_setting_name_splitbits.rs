use splitbits::*;

fn main() {
    splitbits!(pokemon=standard, 0b11011101, "aaabbccc");
}
