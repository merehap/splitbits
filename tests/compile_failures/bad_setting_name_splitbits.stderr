error: proc macro panicked
 --> tests/compile_failures/bad_setting_name_splitbits.rs:4:5
  |
4 |     splitbits!(pokemon=standard, 0b11011101, "aaabbccc");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: assertion `left == right` failed: Only 'min' is allowed as a setting.
            left: "pokemon"
           right: "min"
